deploy/eck-stack/charts/eck-kibana/templates/kibana.yaml

```diff
@@ -19,9 +19,7 @@ spec:
   {{- with or ((.Values.spec).image) (.Values.image) }}
   image: {{ . }}
   {{- end }}
-  {{- with or ((.Values.spec).count) (.Values.count) }}
-  count: {{ . }}
-  {{- end }}
+  count: {{ or ((.Values.spec).count) (.Values.count) | int | default 1 }}
```

Comment:
What is the reasoning for adding a default value in the template? Is the default value defined in the default values.yaml not enough?

Reply:
Ok, I was being overly cautious. I reverted this.